src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -44,6 +44,9 @@ class DefaultStorageKeyAllocator
     DefaultStorageKeyAllocator() = default;
     const char * KeyName() { return mKeyName; }
 
+    // Fabric List
+    const char * StoredDataList() { return SetConst("g/gsdl"); }
```

Comment:
What does `StoredDataList` mean? The comment of `Fabric List` does not seem to help much.

Should this be maybe named `StorageKeyName` or similar?

Reply:
This was changed back to FabricList